Bottom: 467e4721e4eff0a1ec4244d9ec0a6e1b34ffe500
Top:    76f46ccf72183083ecbd5bec0f830269b11780bf
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-09-10 17:10:54 +0100

Refresh of add-routines-to-render-just-a-

---

diff --git a/src/hid/gtk/gtkhid-gl.c b/src/hid/gtk/gtkhid-gl.c
index 588106a..c99459b 100644
--- a/src/hid/gtk/gtkhid-gl.c
+++ b/src/hid/gtk/gtkhid-gl.c
@@ -1549,12 +1549,12 @@ ghid_draw_everything (BoxTypePtr drawn_area)
     reverse_layers = !reverse_layers;
 
   memset (do_group, 0, sizeof (do_group));
-  for (ngroups = 0, i = 0; i < max_group; i++) {
+  for (ngroups = 0, i = 0; i < max_copper_layer; i++) {
     LayerType *l;
     int group;
     int orderi;
 
-    orderi = reverse_layers ? max_group - i - 1 : i;
+    orderi = reverse_layers ? max_copper_layer - i - 1 : i;
 
     // Draw in numerical order when in 3D view
     l = global_view_2d ? LAYER_ON_STACK (i) : LAYER_PTR (orderi);
